python-flask-server/templates/index.html

```diff
@@ -0,0 +1,43 @@
+<!DOCTYPE HTML>
```

Comment:
* should reference where we're getting this from
* anyway to make this not in templates?